smoke-tests/apps/JmxMetric/build.gradle.kts

```diff
@@ -10,4 +10,9 @@ dependencies {
   implementation("org.reactivestreams:reactive-streams:1.0.3")
 
   implementation("org.weakref:jmxutils:1.22")
+
+  // spring modules
+  smokeTestImplementation("org.mock-server:mockserver-netty:5.15.0:shaded")
+  smokeTestImplementation("org.awaitility:awaitility:4.2.0")
+  smokeTestImplementation("io.opentelemetry.proto:opentelemetry-proto:0.14.0-alpha")
```

Comment:
I found I needed these for the HttpRequest, Metric, await to compile in JmsMetricTest.java

Reply:
you don't need mockserver, that is used in the test framework. try to remove it one by one, if your test is green without it, it's safe to remove then.

Reply:
I tried to remove the mockserver and the HttpRequest / request() did not resolve in JmxMetricTest.java. I'm going to keep this there for now unless there is something else I can import that would allow this to compile properly.

Reply:
no problem